test/functionalTest/cases/3122_location_metadata_issues/get_location_attr_with_V2_creation.test

```diff
@@ -0,0 +1,129 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This covers:

![imagen](https://user-images.githubusercontent.com/1534240/48832492-8bf95880-ed79-11e8-8a5d-4d042cd084bb.png)

Great!

NTC

Reply:
It's ok

Reply:
Looking this into detail, entity has been created with geo:xxx in NGSIv1, so it has a "location" GeoJson field. Thus, in fact I think the second condition on the right is "Yes" in this case. Actually it is:

![imagen](https://user-images.githubusercontent.com/1534240/48857346-ea91f700-edb8-11e8-8471-bd1f9d1ba0d8.png)